src/Microsoft.AspNetCore.Server.IISIntegration/NativeMethods.cs

```diff
@@ -207,7 +223,8 @@ public static void HttpEnableWebsockets(IntPtr pInProcessHandler)
 
         public static void HttpCancelIO(IntPtr pInProcessHandler)
         {
-            Validate(http_cancel_io(pInProcessHandler));
+            //TODO: Validation is disabled here because it randomly returns 0x80070490 Element not found
+            http_cancel_io(pInProcessHandler);
```

Comment:
@pakrym my guess is this returns an HRESULT if there is nothing to cancel.

Reply:
Aren't we keeping track of when we are reading?

Reply:
Yes, but we don't know what the native state of reading is. If a read completed in native and is propagating up to managed, in managed we still think the reading is pending. If we call CancelIo, it returns a failure saying there is nothing to cancel. The read will still complete correctly though.

Reply:
True, I'll verify and add it back then